src/test/resources/features/complaints/ComplaintsRegistration.feature

```diff
@@ -108,6 +108,23 @@ Feature: Registration
 
 #     SMC COMPLAINTS
 
+  @ComplaintsRegression @ComplaintsWorkflow
+  Scenario: User is able to complete the Registration stage for an SMC complaint case
+    Given I am logged into "CS" as user "SMC_USER"
+    And I create a single "SMC" case
+    And I allocate the case to myself via the successful case creation screen
+    And I add a "Complainant" correspondent
+    And I click the "Continue" button
+    And I enter the Complainant Details
+    And I enter the complaint details on the Complaint Input page
+    And I click the "Continue" button
+    And I select a "Serious" Complaint Category
+    And I select a Owning CSU
+    And I click the "Finish" button
+    Then the case should be moved to the "Triage" stage
+    And the summary should display the owning team as "Serious Misconduct"
```

Comment:
is there a reason not to include the case details accordion assertion step in this scenario?

Reply:
I included it initially but the step regularly failed due to missing information in the Registration accordion. I realise omitting it is not best practice so have added it back in, but have commented the step out for the time being whilst I investigate the issue.

Reply:
Ideal. Lets merge this for now, and then investigate if its an issue with the test code or if its a bug in DECS.